pkg/segment/query/processor/queryprocessor.go

```diff
@@ -265,7 +267,7 @@ func (qp *QueryProcessor) GetFullResult() (*structs.PipeSearchResponseOuter, err
 // 3. Read from the update channel and the final result channel.
 //
 // Once the final result is sent, no more updates will be sent.
-func (qp *QueryProcessor) GetStreamedResult(stateChan chan *query.QueryStateChanData) error {
+func (qp *QueryProcessor) GetStreamedResult(stateChan chan *query.QueryStateChanData, includeNulls bool) error {
```

Comment:
nit: you already added `includeNulls` as a field on `QueryProcessor`, so you probably shouldn't pass it as a parameter here; use `qp.includeNulls` instead